### DISCUSSION TITLE
Better description of Foxglove WebSockets

### DISCUSSION THREAD

**Discussion by WingRS** [foxglove/community#1016] [Q&A]
Hi!
I want to use Foxglove for our remote debugging, and I wonder if this is the answer for us.
I have a robot (ROS 1) with an internet connection in another city, and we would like to get data from Foxglove studio.
I've installed everything (the ros-noetic-foxglove-bridge. It launches just fine, but what are the next steps?
I want to stream the usual ros msgs, like Vecto3, or Odometry message, how should I proceed?
Thanks

**Comment by banisadr** [comment]
If you have live visualization working over the Foxglove bridge for a local network, the remaining work is networking which isn't Foxglove-specific so there are quite a few resources and options available to you.

Depending on your requirements and constraints, probably the best option is to setup a VPN so your robot and computer are on the same virtual network.

**Comment by WingRS** [comment]
I've gone through it with your support over email. 
What is lacking is a video or a guide on how it works because I've got it done, but it's quite painful as I was trying to understand if it will all work.
In the video/guide, it would be nice to have screenshots of the setup, network description, and, notably - the encryption description.
It's very hard to grasp right now.
